HOWTO.md

```diff
@@ -10,9 +10,7 @@ Welcome to Free-Programming-Books! We welcome new contributors; even those makin
 * [Youtube - Markdown Crash Course](https://www.youtube.com/watch?v=HUBNt18RFbo)
 
 
-Don't hesitate to ask questions; every contributor started with a first PR. You could be our thousandth! Just to see the Contributor Over Time!
-
-![Contributor over time](https://contributor-graph-api.apiseven.com/contributors-svg?repo=EbookFoundation/free-programming-books)
+Don't hesitate to ask questions; every contributor started with a first PR. Join a team of [two thousand contributors!](https://contributor-graph-api.apiseven.com/contributors-svg?repo=EbookFoundation/free-programming-books) 
```

Comment:
And if we avoid depend about translations when counter increments. A text like:

`So... how about becoming part of this booming community?`

Or/and link as a badge in README #5983 

Other graphs about contributors: https://github.com/EbookFoundation/free-programming-books/graphs/contributors